src/python/zquantum/qaoa/ansatzes/warm_start_ansatz_test.py

```diff
@@ -0,0 +1,201 @@
+from zquantum.core.interfaces.ansatz_test import AnsatzTests
+from zquantum.core.circuit import Circuit, Gate, Qubit
+from zquantum.core.utils import compare_unitary
+from zquantum.core.openfermion import change_operator_type
+from .warm_start_ansatz import WarmStartQAOAAnsatz, convert_relaxed_solution_to_angles
+from openfermion import QubitOperator, IsingOperator
+import pytest
+import numpy as np
+import sympy
+
+
+def create_betas(number_of_layers):
+    return list(sympy.symbols(f"beta_:{number_of_layers}"))
+
+
+def create_gammas(number_of_layers):
+    return list(sympy.symbols(f"gamma_:{number_of_layers}"))
```

Comment:
Based on how those functions are used, it seems to me that conversion to list is redundant here.